system/Cookie/Cookie.php

```diff
@@ -96,7 +96,7 @@ class Cookie implements ArrayAccess, CloneableCookieInterface
      * Default attributes for a Cookie object. The keys here are the
      * lowercase attribute names. Do not camelCase!
      *
-     * @var array<string, mixed>
+     * @var array<string, bool|int|object|string>
```

Comment:
Can be `object`?

Reply:
I have some misunderstanding of expires field before, I already remove it.